modules/src/aww.py

```diff
@@ -1,26 +1,28 @@
-import urllib2
-import json
 from random import randint
-
+import praw
 import modules
 from templates.generic import *
 
-headers = {"User-Agent" : "Mozilla/5.0 (Windows NT 6.0; WOW64; rv:24.0) Gecko/20100101 Firefox/24.0"}
-
 #aww pulls top 25 posts from reddit.com/r/aww and serves the receipient a random image
 def process(input, entities):
     output = {}
 
-    request = urllib2.Request('https://www.reddit.com/r/aww/top.json?count=N&after=t3_XXXXX', None, headers)
-    data = urllib2.urlopen(request)
-    json_data = json.load(data)
+    reddit = praw.Reddit(client_id='hidrsV8C-QzMmQ',
+                         client_secret='segvJkUHCUFquxCxqz5ml9-tlmE',
```

Comment:
![](https://memegenerator.net/img/instances/500x/76534678/i-guess-you-shouldnt-commit-secrets-to-github-if-you-dont-want-them-stolen.jpg)

Reply:
Yes I've put some thought into how to handle the secret, the Reddit bot is temporary and can be spun up again.  I figured you would want to generate the secret yourself since you handle deployments.  
What should I do from here?  Would you hold the secret in a system variable that is accessed during production?